src/BlockTypes/ProductAverageRating.php

```diff
@@ -0,0 +1,100 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * ProductAverageRating class.
+ */
+class ProductAverageRating extends AbstractBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-average-rating';
+
+	/**
+	 * API version name.
+	 *
+	 * @var string
+	 */
+	protected $api_version = '2';
+
+	/**
+	 * Get block supports. Shared with the frontend.
+	 * IMPORTANT: If you change anything here, make sure to update the JS file too.
+	 *
+	 * @return array
+	 */
+	protected function get_block_type_supports() {
+		return array(
+			'color'                  =>
+				array(
+					'text'       => true,
+					'background' => true,
+				),
+			'typography'             =>
+				array(
+					'fontSize'                 => true,
+					'__experimentalFontWeight' => true,
+				),
+			'spacing'                =>
+				array(
+					'margin'  => true,
+					'padding' => true,
+				),
+			'__experimentalSelector' => '.wc-block-components-product-average-rating',
+		);
+	}
```

Comment:
I believe you will need to apply the same changes here as well that were made in the `block.json` file.